src/compositor/lipstickcompositorwindow.cpp

```diff
@@ -337,7 +348,6 @@ void LipstickCompositorWindow::handleTouchCancel()
         inputDevice->sendTouchCancelEvent();
         inputDevice->setMouseFocus(0, QPointF());
     }
-    window()->removeEventFilter(this);
```

Comment:
Keeping this line here means that the filter will not be re-added until the next scene change event. Should the filter be installed/removed on visible and enable state changes?